core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1605,17 +1605,114 @@ secret_configs:
         namespace: test-credentials
         name: ci-pull-credentials-pack
         type: kubernetes.io/dockerconfigjson
-  # This secret stores the tokens of the SA which has permissions to push images to the CI central registry
-  # We will remove the one for api.ci after migrating the registry to app.ci
+  # This secret contains a docker config which has the permissions to pull the images from local registry
+  # and push the images to the ci central registry.
   - from:
-      token_image-pusher_api.ci_reg_auth_value.txt:
-        bw_item: build_farm
-        attachment: token_image-pusher_api.ci_reg_auth_value.txt
-      token_image-pusher_app.ci_reg_auth_value.txt:
-        bw_item: build_farm
-        attachment: token_image-pusher_app.ci_reg_auth_value.txt
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: build_farm
+          registry_url_bw_field: ocp3-registry-svc-local-url
```

Comment:
This isn't really you but I still question the decision to make this a bitwarden field. I have no idea what `ocp3-registry-svc-local-url` is and I do not think we have any registry urls that are secret. @droslean can we maybe change this? It would be extremely helpful to keep this readable

Reply:
I tried to convince @droslean but did not make it.
https://coreos.slack.com/archives/GB7NB0CUC/p1602775773264700?thread_ts=1602768947.227900&cid=GB7NB0CUC

:)
Maybe i did not express myself well.

Reply:
@alvaroaleman  @droslean 

At the very least, I can make `registry_url` to provide another way to do it via config.
WDYT?

Reply:
https://github.com/openshift/ci-tools/pull/1314